### DISCUSSION TITLE
Wallet balance not updated

### DISCUSSION THREAD

**Discussion by Hellika22** [Chia-Network/chia-blockchain#15981] [Support]
Hello, why are transactions for 2023 not showing up in Chia Wallet? Accordingly, for this year no coins are visible on the wallet, but a month ago they were still displayed...

**Comment by Hellika22** [comment]
I can provide screenshots of withdrawals from exchanges, help me please

**Comment by BrandtH22** [comment]
Hey @Hellika22 , as a heads up the majority of our support efforts are currently focused on our discord server (https://discord.gg/chia) so please do jump in there for faster and more through responses.

As for some troubleshooting:
- What version are you using and on what operating system?
- Are there any errors in the logs?
- Have you tried resyncing the wallet (in the GUI settings under advanced there is a resync option)
- Were there any notable changes that occurred when the wallet stopped displaying transactions? (version upgrade, power outage, specific coin spends, network changes, etc)
